app/assets/stylesheets/styles.scss

```diff
@@ -61,3 +63,13 @@ th .desc:after {
 #relic {
   color: $relic_color;
 }
+
+.find-panel {
+  background-color: $find_panel_bg;
+  padding: 10px;
+  border-style: solid;
+  border-width: 1px;
+  border-color: $find_panel_border;
```

Comment:
Name of variable `find_panel_border` should be written in all lowercase letters with hyphens instead of underscores